rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackageUtil.groovy

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.rosco.providers.util
+
+import static com.netflix.spinnaker.rosco.providers.util.PackageNameConverter.OsPackageName
+
+interface PackageUtil {
+
+    String getPackageType()
+    String getVersionDelimiter()
+    String getVersionSeparator()
```

Comment:
Any takes on better naming of these two? It's not entirely clear what the difference is. `versionSeperator` is used in `buildAppVersionStr()`, while `versionSeperator` is used in `qualifiedPackageName()`.